challenges/classes.js

```diff
@@ -1,7 +1,50 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+class CuboidMaker {
+  constructor(length, width, height) {
```

Comment:
Its good practice to simply pass one argument rather than length, width and height, something like dimension would have sufficed.